### DISCUSSION TITLE
Powershell Icon not showing properly

### DISCUSSION THREAD

**Discussion by KhoaDuong2005** [JanDeDobbeleer/oh-my-posh#4824] [Troubleshoot]
![Screenshot 2024-04-07 155349](https://github.com/JanDeDobbeleer/oh-my-posh/assets/81700421/e72c944b-a2de-4182-84d6-052b8b820379)

I have a problem with my PowerShell icon not showing properly although Terminal works normally, is there any way that I can fix it? I tried to fix it but nothing works out.
The font that I'm currently using is "if_tea.omp.json".
Thank you if anyone can help

**Comment by JanDeDobbeleer** [comment]
You're not using (or configured) a [Nerd Font](https://ohmyposh.dev/docs/installation/fonts). No icons are showing at all.

**Comment by KhoaDuong2005** [comment]
it showed "not implemented" when im trying to configure the font.

**Reply by JanDeDobbeleer** [reply]
I don't understand what you mean, do you have a screenshot?
